docs/docs/api/index.md

```diff
@@ -0,0 +1,117 @@
+
+- [:octicons-link-external-16: SwaggerUI](swagger.md) (Interactive SwaggerUI
+  documentation)
+- [:octicons-link-external-16: ReDoc](redoc.md) (Interactive ReDoc
+  documentation)
+- [:octicons-link-external-16: OpenAPI Specification, formerly Swagger Specification](openapi.md)
+  (JSON)
+
+<!-- prettier-ignore -->
+!!! note
+    The API documentation is only available via the documentation
+    of Collaboration Manager instances. Only in this case you'll be able
+    to open the links above. You'll see the API documentation matching the
+    version of the Collaboration Manager instance you are using.
+
+    It is not available on:
+
+    - GitHub Pages
+    - Local non-cluster development environments
+    - ReadTheDocs
+
+## Authentication
+
+To authenticate against the API you can use Personal Access Tokens (PAT).
+
+### PAT Creation
+
+To create a personal access token (PAT) you can go to `Menu` > `Tokens`, insert
+a short description and pick a date until when the token should be valid.
+
+<!-- prettier-ignore -->
+!!! info
+    The token which is generated will disappear after leaving the page.
+    Make sure you save it - you won't be able to access it again.
+
+### PAT Scope
+
+Personal access token have the same scope as the user who created it. It is
+therefore important that you never pass on the token and treat it responsibly.
+All requests are made in the name of the user who issued the token and are
+logged accordingly. If you feel that the token has been lost, revoke it
+immediately and inform the Systems Engineering Toolchain team.
+
+### Revoke a PAT
+
+In order to revoke a token go to `Menu` > `Token`. There you can see a list of
+all tokens that belong to your user. By clicking on the delete button, you can
+delete a token, which will no longer be valid for authentication.
+
+### PAT Usage
+
+You can use the token for basic authentication against the API. Use the token
+as password and your username as username.
+
+#### Example with Python
+
+```py
+import requests
+
+base_url = "example.com" # Replace with your base URL
+
+requests.get(
+    f"https://{base_url}/api/v1/projects",
+    auth=("username", "token")
+)
+```
+
+#### Example with cURL
+
+With `cURL` you can use the following command to fetch the list of projects:
+
+```zsh
+curl -u [username]:[token] https://[baseURL]/api/v1/projects
+```
+
+#### Example with `capellambse`
+
+Another example is working with the diagram cache of py-capellambse. The
+implementation of the capella modelling tool `capellambse` uses Python and lets
+you read and write models. For more information have a look at the
+[documentation](https://dsd-dbs.github.io/py-capellambse/) or the
+[Github repository](https://github.com/DSD-DBS/py-capellambse).
+
+```python
+model = capellambse.model.MelodyModel(
+  path="<path to the model on your machine>",
+  diagram_cache={
+    "path": "https://<your backend url>/api/v1/projects/<your project slug>/models/<your model slug>/diagrams/%s",
```

Comment:
I actually don't know any conventions for replacement text in code snippets across various languages, but I noticed you use square brackets in the curl example ([username]) and angled brackets in the Python example (<username>). If that was intentional, then ignore this!

Reply:
In Python, it's common to use braces since that's the format for f-strings. Updated the capellambse related code snippet to match the first one.
In CURL, I'd rather use environment variable. Changed it.